File: VerKdag.cs
using System;

internal class Program
{
    private static void Main(string[] args)
    {   //vraag om de geboorte datum - user input
        Console.Write("Wat is je geboorte datum in maand/dag/jaar: ");
        
        //verkregen verjaardag
        var dataVerjaardag = Console.ReadLine();
        var verjaardag = DateTime.Parse(dataVerjaardag);
        
        //verkrijgen van de datum heden
        var dateNu = DateTime.Now;

        //hoeveel dagen leeft iemand
        TimeSpan verschil = (dateNu - verjaardag);
        var dagen = verschil.Days;
        
        //resten berekenen
        var x = (dagen % 1000);
        
        //tellen hoeveel dagen tot de volgende verKdag
        var Count = 0;

        while(x < 1000) {
            x += 1;
            Count += 1;
        }
        
        //andere naam geven
        dagen = Count;
        
        //Het optellen van de datum plus aantal dagen tot verKdag
        DateTime b = dateNu.AddDays(dagen);

        Console.Write($"Over {dagen} dagen op {b} is je verKdag!");

    }
}

File: verKdag.py
#Zilon Huang & Joran Middelman

#import van library
from datetime import datetime, date, timedelta

#vraag om de geboorte datum - user input
Verjaardag = input("Geef je geboortedatum in dag/maand/jaar: ")

#verkregen verjaardag. Onnodige informatie zoals tijd weghalen
dataVerjaardag = datetime.strptime(Verjaardag, '%d/%m/%Y').date()

#verkrijgen van de datum heden
dataNu = datetime.date(datetime.now())

#hoeveel dagen leeft iemand
data_verschil = (dataNu - dataVerjaardag).days

#andere naam geven
a = data_verschil

#resten berekenen
x = (a % 1000)

#tellen hoeveel dagen tot de volgende verKdag
count = 0

while (x < 1000):
  x += 1
  count += 1

a = count

#het optellen van de datum vandaag plus aantal dagen tot verKdag
b = dataNu + timedelta(days=a)

print(f" Je volgende VerKdag is in {a} dagen!, dat is op {b}")